resources/server_install.rb

```diff
@@ -24,7 +24,7 @@
 property :extconf_directory, String,        default: lazy { ext_conf_dir }
 property :data_directory,    String,        default: lazy { data_dir }
 property :external_pid_file, String,        default: lazy { "/var/run/mysql/#{version}-main.pid" }
-property :password,          [String, nil], default: 'generate'
+property :password,          [String, nil], default: lazy { node['mariadb']['server_root_password'] }, sensitive: true
```

Comment:
Hi,

Why the removal of generate, it would be better to fix this behaviour as otherwise this would be a breaking change?

Also we are actively moving away from attributes and do not want to introduce new attributes,

Thanks 👍